datafusion/proto/tests/cases/roundtrip_physical_plan.rs

```diff
@@ -374,18 +374,24 @@ fn roundtrip_aggregate_udaf() -> Result<()> {
         }
     }
 
-    let rt_func: ReturnTypeFunction = Arc::new(move |_| Ok(Arc::new(DataType::Int64)));
+    let return_type = DataType::Int64;
     let accumulator: AccumulatorFactoryFunction = Arc::new(|_| Ok(Box::new(Example)));
-    let st_func: StateTypeFunction =
-        Arc::new(move |_| Ok(Arc::new(vec![DataType::Int64])));
-
-    let udaf = AggregateUDF::new(
-        "example",
-        &Signature::exact(vec![DataType::Int64], Volatility::Immutable),
-        &rt_func,
-        &accumulator,
-        &st_func,
-    );
+    let state_type = vec![DataType::Int64];
+
+    // let udaf = AggregateUDF::new(
```

Comment:
Did you mean to leave this commented out?

Reply:
I forget delete this old code 😅